### ISSUE TITLE
Better control of notifications

### ISSUE THREAD

**Issue by juanvelascogomez** [captainGeech42/ransomwatch#48]
After using ransomwatch for a couple of days, maybe it's a good idea to let users control which notifications they want to receive (e.g., not receiving the site-down, site-up on every iteration, just at the end of the day or logging control)

This recommendation is produced by having in my case a frequent crontab (every hour) so that the notifications of new victims is fast, but without the need for generic information such as errors and status of the hidden service to be sent so frequently.

What do you think?

**Comment by captainGeech42** [comment]
I agree that the existing controls available for notifications aren't sufficient. This is mostly being tracked in #26
